task2/task2_solution.py

```diff
@@ -0,0 +1,110 @@
+# NOTES:
+#
+# no idea whether it's ok to make
+# to_letters function a method of the class
+# (staticmethod doesn't work)
+# external func works fine, let is be so
```

Comment:
As it's part of Journal class it is better for it to be as method, not some separate function